nautobot/dcim/tests/integration/test_create_device.py

```diff
@@ -0,0 +1,140 @@
+from django.urls import reverse
+from selenium.webdriver.common.keys import Keys
+
+from nautobot.core.testing.integration import SeleniumTestCase
+
+
+class CreateDeviceTestCase(SeleniumTestCase):
+    """
+    Create a device and all pre-requisite objects through the UI.
+    """
+
+    def _click_nav_bar(self, parent_menu_name, child_menu_name):
```

Comment:
Might make sense for this (and maybe some of the other helpers below) to be added to `SeleniumTestCase` so that we can later refactor some of the other integration tests to use them?

Reply:
That sounds good to me. I'm happy with how these helper functions turned out